analyses/molecular-subtyping-embryonal/04-table-prep.Rmd

```diff
@@ -507,6 +511,27 @@ all_data_df %>%
   filter(CIC_fusions != "None")
 ```
 
+### CNS Neuroblastoma
+
+> * CNS Neuroblastoma tumor 
+> * Primary site does not contain "Other locations NOS"
+> * Pathology free text diagnosis does not contain "peripheral"
+
+```{r}
+neuroblastoma_df <- all_data_df %>%
+  filter(
+    pathology_diagnosis == "Neuroblastoma",
```

Comment:
I also want to point out the difference in logic here in `04-table-prep.Rmd` from the `01` notebook. This logic excludes the `Ganglioneuroblastoma` sample that was included in the `01` notebook [per this comment](https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/788#issuecomment-700717514), which states that `Neuroblastoma` samples that meet this logic should be classified as `CNS Neuroblastoma` (which is being done here), while the `Ganglioneuroblastoma` samples should be designated as `Embryonal NOS` for now due to being characterized histologically without clear molecular lesions.